app/styles/custom-components/_vlc-document.scss

```diff
@@ -0,0 +1,3 @@
+.vlc-document__icon::before {
+  color: green;
```

Comment:
I think the Figma makes this green specifically `#8BAE00`.

Reply:
Dit lijkt nog niet in orde?